LOG.tbasicu

```diff
@@ -124,7 +125,7 @@ Type tLogThis
     me.TagAlert = "[ALERT]"
     me.TagPanic = "[EMERGENCY]"
     me.minSeverityHighlight = %TAG_ERR
-    me.msgboxStyle = %MB_OK + %MB_TOPMOST ' %MB_ICONINFORMATION
+    me.msgboxStyle = %MB_OK | %MB_TOPMOST ' %MB_ICONINFORMATION
```

Comment:
as for unifying _+_ and _or_ to _ors_,  why a _|_ here ?

Reply:
I simply forgot, fixed :)

Reply:
ok, seen !